### DISCUSSION TITLE
在 antv-dagre 布局下使用 orth router 的边如何避开节点？

### DISCUSSION THREAD

**Discussion by Cay222** [antvis/G6#6437] [Q&A]
[运行代码的链接](https://codesandbox.io/p/sandbox/avoid-node-3n46c8)
想要实现红色箭头的效果：
![image](https://github.com/user-attachments/assets/1d9c47d4-80a3-4ec5-9502-d76961aad8bd)

**Comment by yvonneyx** [comment]
提供一个思路，可以自定义一个后处理

使用 dagre 布局后，算法会输出节点的位置与边的控制点（此处为控制点 1 和控制点 2）。如果想要使用正交边进行连线，情况会比较简单，只存在拐点 1 和拐点 2 两种情况，再进行碰撞检测并选择适合的拐点即可

<img width="408" alt="image" src="https://github.com/user-attachments/assets/2c3076a0-ab66-4598-8c76-fcd0808474d1">
